src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
```

Comment:
According to the checklist, we should use abstract references instead of specific ones where possible. Instead of `Machine truck = new Truck();`, consider using `Truck truck = new Truck();` since we are not utilizing polymorphism here.